deployment/openshift/about.md

```diff
@@ -1,12 +1,168 @@
 ---
-label: need-help
-
 title: OpenShift
 subsection: openshift
 section: deployment
-description:
+description: OpenShift
 ---
 
 # {{page.title}}
 
-Platform as a Service (Paas) cloud to host autoscaling network applications.
+OpenShift Origin is a distribution of [Kubernetes](https://kubernetes.io) optimized for continuous application development and multi-tenant deployment. It adds developer and operations-centric tools on top of Kubernetes to enable rapid application development, easy deployment and scaling, and long-term lifecycle maintenance for small and large teams.
+
+Features:
+
+* Easily build applications with integrated service discovery and persistent storage.
+* Quickly and easily scale applications to handle periods of increased demand.
+  * Support for automatic high availability, load balancing, health checking, and failover.
+* Push source code to your Git repository and automatically deploy containerized applications.
+* Web console and command-line client for building and monitoring applications.
+* Centralized administration and management of an entire stack, team, or organization.
+  * Create reusable templates for components of your system, and iteratively deploy them over time.
+  * Roll out modifications to software stacks to your entire organization in a controlled fashion.
+  * Integration with your existing authentication mechanisms, including LDAP, Active Directory, and public OAuth providers such as GitHub.
+* Multi-tenancy support, including team and user isolation of containers, builds, and network communication.
+  * Allow developers to run containers securely with fine-grained controls in production.
+  * Limit, track, and manage the developers and teams on the platform.
+* Integrated Docker registry, automatic edge load balancing, cluster logging, and integrated metrics.
+
+## Start a local OpenShift all-in-one cluster
+
+The quickest way to try out OpenShift on your own computer is using the `oc cluster up` command.
+First, install the dependencies. We'll need [Docker](/tools/docker/docker-installation.html) and the `origin-clients` that provides the `oc` binary.
+
+```console
+$ sudo dnf install origin-clients docker
+```
+
+*Note*: we recommend following the steps in the [Docker installation page](/tools/docker/docker-installation.html#why-cant-i-use-docker-command-as-a-non-root-user-by-default) to add your user to the `docker` group, so that you can run the `docker` command without `sudo`.
```

Comment:
@sferich888 Please check docker installation instruction and feel free to update it. We can use two ways how to work/play with docker.